Apps/Playground/Scripts/experience.js

```diff
@@ -24,6 +24,47 @@ function CreateBoxAsync(scene) {
     return Promise.resolve();
 }
 
+function CreateMultiCameraView(scene) {
```

Comment:
Is this change intended? Maybe we can create a playground for this and use validation tests?

Reply:
No, changed to experience.js were not intended. Just reverted it to match master.